core/src/main/java/io/parsingdata/metal/token/Cho.java

```diff
@@ -27,31 +27,31 @@
 
 public class Cho extends Token {
 
-    private final Token[] _tokens;
+    public final Token[] tokens;
```

Comment:
An array is mutable. We should expose a clone of the array, or a collection:

```java
public List<Token> tokens() {
    return Arrays.asList(tokens);
}
```

(Same holds for `Seq.)